public/dist/style.css

```diff
@@ -580,3 +626,7 @@ a:hover {
   padding-bottom: 20px;
   font-weight: 500;
 }
+
```

Comment:
This will affect my ratings as well I believe. Might want to keep that in mind when we deploy. (This will add margin-left -3px to all of my stars).